src/libraries/System.Security.Cryptography/tests/X509Certificates/ExportTests.cs

```diff
@@ -834,7 +834,7 @@ internal static (int certs, int keys) VerifyPkcs12(
                         AssertEncryptionAlgorithm(epki.EncryptionAlgorithm);
                         keys++;
                     }
-                    else if (safeBag is Pkcs12CertBag && wasEncryptedSafe)
+                    else if (safeBag is Pkcs12CertBag && (wasEncryptedSafe || PlatformDetection.IsWindows))
```

Comment:
Since modern Windows is doing it the way the test expects, and I'm pretty sure older Windows (XP, 7, 8, 8.1) do it the way the test expects... maybe we want a more limited `||` here (Windows 10 before RS2?)

Since Windows is a black box, I'm sort of OK with this here... but at the same time it seems like this is what would tell us if Windows changes how they do PFX in the future, and we would then want to be informed.  Maybe that means a different test elsewhere with the relaxed condition here?

Reply:
Okay, after digging through some offline conversations, we did find that RS2 is where certs went to being encrypted, so I changed the counter to be more picky about the version range.